docs_src/custom_response/tutorial008.py

```diff
@@ -7,5 +7,8 @@
 
 @app.get("/")
 def main():
-    file_like = open(some_file_path, mode="rb")
```

Comment:
@tiangolo pytest 6.2 includes support for a new plugin [unraisableexception](https://docs.pytest.org/en/stable/usage.html#warning-about-unraisable-exceptions-and-unhandled-thread-exceptions) that can be used to detect the ResourceWarnings raised when files are garbage collected without being closed explicitly

Reply:
Get it, thanks! I think I'll have to update something in the docs to account for these changes, as this file is injected in the examples. But I can do that later, for now we can check the other questions I have.

Reply:
I'm going to change how StreamingResponse works - and make it take an async context manager factory instead, deprecating the old aiter parameter

Reply:
so it would work like this instead:

```python
return StreamingResponse(iterfile, media_type="video/mp4")
```